lib/api/model/model.dart

```diff
@@ -801,4 +835,4 @@ class DmMessage extends Message {
 
   @override
   Map<String, dynamic> toJson() => _$DmMessageToJson(this);
-}
+}
```

Comment:
This final line is incomplete. See https://github.com/zulip/zulip-flutter/pull/534#discussion_r1515353889 for discussion.